root/static/scripts/edit/components/DateRangeFieldset.js

```diff
@@ -205,15 +207,18 @@ component _DateRangeFieldset(
         field={field}
         includeSubFields={false}
       />
-      <FormRowCheckbox
-        disabled={
-          disabled ||
-          !isDateEmpty(partialDateFromField(subfields.end_date))
-        }
-        field={subfields.ended}
-        label={endedLabel}
-        onChange={hooks.handleEndedChange}
-      />
+      {hideEnded ? null : (
+        <FormRowCheckbox
+          disabled={
+            disabled ||
+            !isDateEmpty(partialDateFromField(subfields.end_date))
+          }
+          field={subfields.ended}
+          label={endedLabel ?? l('Ended')}
```

Comment:
This would be a new string. Context is just lacking to match the existing string:
```suggestion
          label={endedLabel ?? lp('Ended', 'label ended boolean')}
```

Reply:
No, this is not a MB label. This is anything-with-dates ended (label here is the *HTML* label). That said, that's not great for a context... we *could* actually have a label picker that looks at entity type and picks the right context?

Reply:
Or just keep the component’s property `endedLabel` mandatory instead of making it optional?